LetsDoStuff.WebApi/Program.cs

```diff
@@ -19,70 +22,94 @@ public static void Main()
-                .Options;
-
-            SeedTestData(options);
-
             var host = new WebHostBuilder()
                 .UseKestrel()
-                .ConfigureAppConfiguration(
-                    (host, configurationBuilder) =>
-                    {
-                        configurationBuilder.AddJsonFile("appsettings.json");
-                    })
+                .ConfigureAppConfiguration((host, configurationBuilder) =>
+                {
+                  configurationBuilder.AddJsonFile("appsettings.json");
+                })
+                .ConfigureLogging((context, logging) =>
+                {
+                    var env = context.HostingEnvironment;
+                    var config = context.Configuration.GetSection("Logging");
+                    logging.AddConfiguration(config);
+                    logging.AddConsole();
+                    logging.AddFilter("Microsoft.EntityFrameworkCore.Database.Command", LogLevel.Critical)
+                    .AddFilter("Default", LogLevel.Trace)
+                    .AddFilter("Microsoft", LogLevel.Warning)
+                    .AddFilter("System", LogLevel.Information)
+                    .AddFilter("Microsoft.AspNetCore.Authentication", LogLevel.Warning);
+                })
                 .UseStartup<Startup>()
                 .UseUrls(baseUrl)
                 .Build();
 
+            using (var scope = host.Services.CreateScope())
+            {
+                var services = scope.ServiceProvider;
+                var context = services.GetService<LdsContext>();
+                var userService = services.GetService<IUserService>();
+                SeedTestData(context, userService);
+            }
+
             host.Run();
         }
 
-        private static void SeedTestData(DbContextOptions<LdsContext> options)
+        private static void SeedTestData(LdsContext context, IUserService userService)
         {
-            using (LdsContext context = new LdsContext(options))
+            context.Database.EnsureCreated();
+            if (!context.Users.Any())
             {
-                context.Database.EnsureCreated();
-
-                if (!context.Users.Any())
+                userService.RegisterUser(new RegisterRequest()
                 {
-                    context.Users.Add(new User { Name = "Tom", Login = "Tom1", Password = "1234", Age = 33, Role = "admin" });
-                    context.Users.Add(new User { Name = "Alice", Login = "Alice1", Password = "0000", Age = 26, Role = "user" });
-                    context.SaveChanges();
-                }
-
-                if (!context.Tags.Any())
+                    FirstName = "Roman",
+                    LastName = "Onofreichuk",
+                    Email = "test@gmail.com",
+                    Password = "12test",
+                    ConfirmPassword = "12test"
+                });
+                userService.RegisterUser(new RegisterRequest()
```

Comment:
Let's not use services in seed methods. We are going to make it in a separate project in the future